py/desispec/io/photo.py

```diff
@@ -521,8 +521,8 @@ def gather_targetphot(input_cat, photocache=None, racolumn='TARGET_RA',
     
     return out
 
-def tractorphot_datamodel(from_file=False):
-    """Initialize the tractorphot data model for DR9 photometry.
+def tractorphot_datamodel(from_file=False, dr10=False):
```

Comment:
Let's make this new option more generic to anticipate a future with dr11, dr12, etc.  e.g. make the option `datarelease='dr9'` by default, document which options are currently supported, and raise an exception if some other dr is given.  But don't bake in dr10 as the one-and-only alternative to dr9.  This is in the same spirit as elsewhere changing to `legacysurveydir` instead of `dr9dir` (and not `dr9or10dir`).

Reply:
Done in https://github.com/desihub/desispec/pull/2079/commits/24900476cda7c0811421354f78129c91ddb4b3a0.